### PR TITLE
Delete algorithms and cipher checking

### PR DESCRIPTION
**Pull request by wsargent**
JDK 1.8 and JDK 11 have much better security now and these can be removed.

### CODE REVIEW

**Comment by mkurz** [comment]
@wsargent the build fails.

**Comment by wsargent** [comment]
@mkurz Running into the same race condition...

**Comment by wsargent** [comment]
@mkurz  fixed.  I added the workaround as a different PR https://github.com/lightbend/ssl-config/pull/306

File: ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/Config.scala [hunk 1]
```diff
@@ -37,7 +37,7 @@ final class KeyStoreConfig private[sslconfig] (
     val storeType: String = KeyStore.getDefaultType) {
   assert(filePath.isDefined ^ data.isDefined, "Either key store path or data must be defined, but not both.")
 
-  /** Disables `filePath` – only one of those can be used at any given time. */
+  /** Disables `filePath` – only one of those can be used at any given time.com.typesafe.sslconfig.ssl.ConfigSSLContextBuilderSpec */
```
Comment (mkurz) [inline ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/Config.scala:40]: @wsargent Seems like an unintended copy/paste?

**Comment by SethTisue** [comment]
@wsargent ping

**Comment by wsargent** [comment]
ack

**Comment by SethTisue** [comment]
thanks Will!

**Pull request merged**